### ISSUE TITLE
Putting a `className` in subfeatures prevents from zooming

### ISSUE THREAD

**Issue by althonos** [BioComputingUP/feature-viewer-typescript#7]
Hi there, first of all thanks for this library it makes it much easier for us to visualize some annotations in metagenomic contigs.

I noticed the following bug while trying to apply a `className` to some subfeatures that you were not able to zoom that feature anymore; all other features would scale but the one you gave a custom class name. (For the record, zooming work if you close the subfeature, zoom, and open it back again, but with the feature being open no domain on that feature will scale).

I used the latest version of the library from `npm`, with the following code:
```javascript
var subfeatures = [{
    "type": "rect",
    "id": "domainsPfam",
    "label": "Pfam",
    "className": "domain",
     "data": [ { "x": ..., "y": ..., "tooltip": ..., "label": ..., "color": ... }, ...],
}]

var features = [{
        "id": "domains",
        "label": "Domains",
        "type": "rect",
        "data": [],
        "isOpen": False,
        "subfeatures": subfeatures,
}];

fv.addFeatures(features);
```

**Comment by Lisanna** [comment]
Thank you, it's a pleasure to know that you are using it! 
And thank you for reporting the issue, it should be solved now in version 2.1.7.

**Comment by althonos** [comment]
@Lisanna : would you mind having a look at #8 as well ? :smiley:
